app/org/maproulette/controllers/api/TaskController.scala

```diff
@@ -538,6 +538,51 @@ class TaskController @Inject() (
     }
   }
 
+  /**
+    * This function will set the review status to "Unnecessary", essentially removing the
+    * review request.
+    *
+    * User must have write access to parent challenge(s).
+    *
+    * @param ids The ids of the tasks to update
+    * @return The number of tasks updated.
+    */
+  def removeReviewRequest(ids: String): Action[AnyContent] = Action.async { implicit request =>
+    this.sessionManager.authenticatedRequest { implicit user =>
+      SearchParameters.withSearch { p =>
+        var params = p
```

Comment:
You should always try to steer clear of `var` in Scala. I have found that in Scala `var` was more a concession that anything else. It is a language that likes to use immutable objects and only switch to mutable objects when absolutely necessary. In general there is rarely a time when you actually need to use `var`.

Reply:
Ok, I've changed it.